tokio-fs/src/file/mod.rs

```diff
@@ -34,6 +34,46 @@ use std::path::Path;
 /// Files are automatically closed when they go out of scope.
 ///
 /// [std]: https://doc.rust-lang.org/std/fs/struct.File.html
+///
+/// # Examples
+///
+/// Create a new file and asynchronously write bytes to it:
+///
+/// ```no_run
+/// extern crate tokio;
+/// 
+/// use tokio::prelude::*;
```

Comment:
Do you actually need the prelude here?  Given that the File type is fully qualified?